activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -84,6 +84,15 @@ def sql_color(sql)
       def logger
         ActiveRecord::Base.logger
       end
+
+      def debug(progname = nil, &block)
+        backtrace = Rails.backtrace_cleaner.clean caller
```

Comment:
I don't think we should couple Active Record with Railties here. One alternative is define our own cleaner here or doing the same as deprecation is doing https://github.com/rails/rails/blob/master/activesupport/lib/active_support/deprecation/reporting.rb#L82